scripts/tf_cnn_benchmarks/allreduce.py

```diff
@@ -19,6 +19,7 @@
 import collections as pycoll
 import re
 
+from six.moves import xrange
```

Comment:
The linter is complaining about redefining a builtin. Add `  # pylint: disable=redefined-builtin` to the end of this line, as in [benchmark_cnn.py](https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/benchmark_cnn.py#L34). Make sure there are two spaces between xrange and #